docs/installing_wash.md

```diff
@@ -4,17 +4,19 @@ title: Installing Wash
 Wash is distributed as a single binary; the only prerequisite is `libfuse`. Here’s how to install it.
 
 * Install `libfuse` if you haven’t already
-    * E.g. on MacOS using homebrew: `brew cask install osxfuse`
-    * E.g. on CentOS: `yum install fuse fuse-libs`
-    * E.g. on Debian/Ubuntu: `apt-get install fuse`
+    * On MacOS using homebrew: `brew cask install osxfuse`
+        * You'll also need to restart your computer
+    * On CentOS: `yum install fuse fuse-libs`
+    * On Debian/Ubuntu: `apt-get install fuse`
 
 * Install the Wash binary
     * E.g. on MacOS using homebrew: `brew install puppetlabs/puppet/wash`
```

Comment:
Want to drop `E.g.` here like you did earlier?

Reply:
Thanks, done.